src/cpp/fastdds/builtin/type_lookup_service/TypeLookupRequestListener.cpp

```diff
@@ -29,6 +26,10 @@
 
 #include <utility>
 
+#include <fastdds/builtin/type_lookup_service/common/TypeLookupTypes.hpp>
+#include <fastdds/builtin/type_lookup_service/TypeLookupRequestListener.hpp>
+#include <fastdds/builtin/type_lookup_service/TypeLookupManager.hpp>
```

Comment:
Please, apply recommended include order for every header included in this file:

1. Header declaring the implemented class. Blank line.
2. C headers (if possible avoid their use including the following). Blank line.
3. C++ headers. Blank line.
4. External header files (external dependencies). Blank line.
5. Public internal headers. Blank line.
6. Private internal headers.